auslib/web/common/releases_spec.yml

```diff
@@ -109,7 +109,7 @@ definitions:
       product:
         type: string
       read_only:
-        type: ['boolean', 'null']
+        type: boolean
```

Comment:
I don't quite understand why adding "null" to most of the properties is necessary. As far as I know, the API always returns them, so why is null valid?